gallery/src/pages/AccessibilityChecklist.tsx

```diff
@@ -0,0 +1,1516 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+import {
+  NxP,
+  NxH3,
+  NxH4,
+  NxList,
+  NxTextLink,
+  NxCheckbox,
+  NxStatefulCheckbox,
+  NxFieldset,
+  useToggle
+} from '@sonatype/react-shared-components';
+
+function Accessibility() {
+
+  const [isA, toggleA] = useToggle(false),
+      [isAA, toggleAA] = useToggle(false),
+      [isAAA, toggleAAA] = useToggle(false),
+      [isReco, toggleReco] = useToggle(false);
+
+  return (
+    <GalleryTile title="WCAG Accessibility UX Checklist" className="nx-tile--accessibility-list">
+      <NxP>
+        Accessibility UX checklist to serve as a method for designers and developers to assess the WCAG compliance of
+        their work.
+      </NxP>
+      <NxP>
+        WCAG 2.1 Verification - 116 Questions | Cumulative list
+      </NxP>
+      <NxFieldset label="Filter by level" isRequired className="gallery-accessibility-filter">
+        <NxCheckbox onChange={toggleA} isChecked={!isA}>Level A</NxCheckbox>
+        <NxCheckbox onChange={toggleAA} isChecked={!isAA}>Level AA</NxCheckbox>
+        <NxCheckbox onChange={toggleAAA} isChecked={!isAAA}>Level AAA</NxCheckbox>
+        <NxCheckbox onChange={toggleReco} isChecked={!isReco}>Recommendations</NxCheckbox>
+      </NxFieldset>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <NxH3>
+            1. Perceivable
+          </NxH3>
+        </header>
+        <NxP>
+          <em>
+            Information and user interface components must be presentable to users in ways they can perceive.
+          </em>
+        </NxP>
+        <NxH4>
+          1.1 Text alternatives- Provide text alternatives for any non-text content so that it can be changed into other
+          forms people need, such as large print, braille, speech, symbols or simpler language.
+        </NxH4>
+        <NxList>
+          <NxList.Item className={`level-a ${isA && 'hidden'}`}>
```

Comment:
This syntax is adding a classname of "false" to each non-hidden list item

Reply:
> This syntax is adding a classname of "false" to each non-hidden list item

Yeah, that's not what I wanted, here is a much better filtering implementation: https://github.com/sonatype/sonatype-react-shared-components/pull/657/commits/c1212110adfdc59921c66c0c91af628da4ab1615